samples/RhythmGame/src/main/cpp/utils/UtilityFunctions.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef RHYTHMGAME_TIMEFUNCTIONS_H
+#define RHYTHMGAME_TIMEFUNCTIONS_H
+
+#include <stdint.h>
+
+
+constexpr int kMillisecondsInSecond = 1000;
+constexpr int kMicrosecondsInMillisecond = 1000;
+constexpr int kNanosecondsInMillisecond = 1000000;
```

Comment:
These constant need to be int64_t otherwise you can easily get overflows during conversions on a 32-bit architecture.

Reply:
Done